src/app/technical-record/technical-record.component.html

```diff
@@ -1,9 +1,7 @@
 <div class="govuk-width-container">
   <a class="govuk-back-link" vtmBackButton>Back</a>
   <main class="govuk-main-wrapper">
-    <div
-      *ngIf="activeRecord.vehicleType == 'hgv' || activeRecord.vehicleType == 'trl'; else ifPSV"
-    >
+    <div>
```

Comment:
Was it a missing <div>?
Not sure why we need to be having an empty div which looks like a wrapper here? it has nothing on it